src/containers/FormikForm/components/FormikMetaImage.jsx

```diff
@@ -71,22 +75,32 @@ const FormikMetaImage = ({
   return (
     <Fragment>
       <div {...formClasses('meta-image')}>
-        <img src={image.imageUrl} alt={alt} />
+        {banner ? (
+          <div
```

Comment:
Kan kanskje løses med emotion. Frontendfolka kan meir om dette.

Reply:
For et eksempel på det kan du f.eks se i `PublishTopic.tsx` (og flere filer). Om det er mer enn en verdi er jeg enig i at det kan flyttes ut, men sånn som det er nå er vel inline like greit? Noen andre som har tanker om det? @brdalsnes

Reply:
Enig med Elena